doc/guides/maintaining-V8.md

```diff
@@ -414,6 +414,11 @@ This would require some tooling to:
 <sup>1</sup>Node.js 0.12 and older are intentionally omitted from this document
 as their support has ended.
 
+<sup>2</sup>At the moment the V8 tests still require Python 2. To run these
+tests locally, you can run `/path/to/python2 ./configure.py` before running
```

Comment:
I'm not sure if it works everywhere, but I've always been able to run `PYTHON2 ./configure.py` instead of providing the entire path.

Reply:
That's true. Providing just the command name is the same as giving the whole path in this case.

I would use lowercase `python2` in the documentation if that's alright.

- I like that all-caps `PYTHON2` makes people think for a moment and pick the actual Python 2 command that's on their system.
- On the other hand, `python2` is probably the right command, if not `python` or `python2.7`, and making people think is a con as well as a pro.

Reply:
I'm going back and forth on all-caps `PYTHON2`, but I actually like it. It reminds people their Python 2 command might not be `python2`, and that whatever it is is okay, so long as it's Python 2 in some shape or form. When I read it, I don't even think about it that hard. It makes intuitive sense.

Leaving a bit of time for others to potentially comment before I submit a commit with those suggested changes.